README.md

```diff
@@ -0,0 +1,122 @@
+[Deployment repository][deployment_url]. Here you will be able to deploy it following some few steps.
+
+## Documentation
+
+For additional documentation, please refer to the [Wiki][UNCode_wiki_url].
+
+## Notes on security
+
+Docker containers can be used securely with SELinux enabled. Please do not run untrusted code without activating SELinux.
+
+## Roadmap
+
+See the [UNCode GitHub Project][project_url] for a list of proposed features, known issues and how they are being 
+tackled.
+
+## Contributing
+
+Go to [CONTRIBUTING][contributing_url] to see the guidelines and how to start contributing to UNCode.
+
+## License
+
+Distributed under the AGPL-3.0 License. See [LICENSE][license_url] for more information.
+
+## Contact
+
+In case of technical questions, please use the [gitter communication channel][gitter_url].
+
+In case you want to host your course on our deployment, email us on: <uncode_fibog@unal.edu.co>
+
+UNCode: <https://uncode.unal.edu.co>
+
+Project page: <https://juezun.github.io/UNCode_page/>
+
+## Publications
+
+- F. Restrepo-Calle, J.J. Ramírez-Echeverry, F.A. Gonzalez. "[UNCode: Interactive system for learning and automatic 
+evaluation of computer programming skills.][uncode_publication_url]". In Proceedings of the 10th annual International 
+Conference on Education  and New Learning Technologies EDULEARN 2018. Palma de Mallorca, Spain, 2nd-4th July 2018, 
+pp. 6888-6898. doi: 10.21125/edulearn.2018.1632.
+
+```
+    @article{Restrepo-Calle2018,
+        title = {{Uncode: Interactive System for Learning and Automatic Evaluation of Computer Programming Skills}},
+        year = {2018},
+        journal = {EDULEARN18 Proceedings},
+        author = {Restrepo-Calle, Felipe and Ram{\'{i}}rez-Echeverry, Jhon Jairo and Gonzalez, Fabio A.},
+        number = {July},
+        pages = {6888--6898},
+        volume = {1},
+        isbn = {9788409027095},
+        doi = {10.21125/edulearn.2018.1632},
+        keywords = {automatic grading tool, computer programming, educational environment}
+    }
+```
+
+
+[uncode_url]: https://uncode.unal.edu.co/courselist
+[license_url]: https://github.com/JuezUN/INGInious/blob/master/LICENSE
+[contributors_url]: https://github.com/JuezUN/INGInious/graphs/contributors
+[issues_url]: https://github.com/JuezUN/INGInious/issues
+[codacy_badge_url]: https://www.codacy.com/gh/JuezUN/INGInious/dashboard?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=JuezUN/INGInious&amp;utm_campaign=Badge_Grade
+[travis_status_url]: https://travis-ci.org/JuezUN/INGInious
+[CLA_url]: https://cla-assistant.io/JuezUN/INGInious
+[gitter_url]:https://gitter.im/uncode-unal/community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge
+[INGInious_repo]: https://github.com/UCL-INGI/INGInious
+[docker_page]: https://www.docker.com/
+[python_url]: https://www.python.org/
+[mongo_url]: https://www.mongodb.com/
+[UNCode_wiki_url]: https://github.com/JuezUN/INGInious/wiki
```

Comment:
Codacy found an issue: [[definition-case] Do not use uppercase characters in definition labels](https://app.codacy.com/gh/JuezUN/INGInious/pullRequest?prid=6256833)